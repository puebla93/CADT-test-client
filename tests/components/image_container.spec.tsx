import { render } from '@testing-library/react'
import ImageContainer from '../../components/image_container'

describe('ImageContainer component tests', () => {
    test('ImageContainer matches snapshot', async () => {
        const {container} = render(
            <ImageContainer></ImageContainer>
        );
        expect(container).toMatchSnapshot();
    });
})
