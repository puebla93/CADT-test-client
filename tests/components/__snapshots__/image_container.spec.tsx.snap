// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageContainer component tests ImageContainer matches snapshot 1`] = `
<div>
  <div
    class="sc-bcXHqe jCZMxz"
  />
</div>
`;
